c_glib/parquet-glib/arrow-file-reader.cpp

```diff
@@ -310,16 +310,16 @@ gparquet_arrow_file_reader_read_column(GParquetArrowFileReader *reader,
     return NULL;
   }
 
-  std::shared_ptr<arrow::Array> arrow_array;
-  status = parquet_arrow_file_reader->ReadColumn(column_index, &arrow_array);
+  std::shared_ptr<arrow::ChunkedArray> chunked_array;
```

Comment:
Can you add `arrow_` prefix to Arrow C++ objects?

Reply:
Oh... thank you for pointing out. I've add `arrow_` prefix.